.rubocop_todo.yml

```diff
@@ -2,17 +2,25 @@ require: rubocop-rspec
 
 Metrics/PerceivedComplexity:
   Exclude:
+    - 'lib/qa/authorities/linked_data/config/config_merge.rb'
```

Comment:
Why are we ignoring these warnings instead of refactoring away the cause of the warning?  Moreover, if ignoring is absolutely required, I'd prefer an inline ignore, so I can see exactly what method is causing this problem.